GRAMMAR.md

```diff
@@ -40,15 +40,17 @@ A `condition` is any expression that evaluates to a result of type boolean.
 (lparen)? \<logical expr> | \<comparison expr> \<logical operator> \<logical expr> |
 \<comparison expr> (\<rparen)?
 
+### symbol
+
+* \<string>
 
 ### string
 
 * \<string>
 
 ### quoted string
 
-A string delimited by quotes. Note: you can use a quoted string whenever you
-need to include logical operators or whitespace in a string value.
+A string delimited by quotes. Supported delimiters: double-quotes
```

Comment:
Mind adding that one can escape double quotes within a quoted string with a backslash? And that a backslash can be escaped with another backslash?

Reply:
ty!